apps/portals/admin/src/components/Layout/Layout.tsx

```diff
@@ -100,7 +100,9 @@ export const Layout: FC = ({ children }) => {
 
   return (
     <LayoutOuterContainer>
-      <LayoutModuleContainer layout={layout}>{children}</LayoutModuleContainer>
+      <LayoutModuleContainer layout={!activeModule ? 'none' : layout}>
```

Comment:
The layout prop should be handled the same when ModuleLayoutWrapper is true'ish (see above).

Maybe you can default `layout` to 'non'` when destructuring?

Reply:
Good point on the former one. But no then we will have to add the `layout: 'default' to every module.`